easybuild/easyconfigs/a/Anaconda/Anaconda-1.8.0-Linux-x86_64.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'Binary'
+
+name = 'Anaconda'
+version = '1.8.0-Linux-x86_64'
```

Comment:
this could perhaps be done via versionsuffix; 
the question here is how you'd wish --try-toolchain-version to behave (should you need to pass all the string or not?)